Sources/DownloadManager/ViewModel/DownloadManager.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  DownloadManager.swift
+//
+//
+//  Created by Abhishek Pandey on 29/08/23.
+//
+
+import Foundation
+import SwiftUI
+
+public class DownloadManager: ObservableObject {
+    
+    //MARK: - properties
+    var playerManager = PlayerManager()
```

Comment:
I think this should be a constant, I can be wrong, so please go through this once.

Reply:
Updated var to let